cmd/access/node_builder/staked_access_node_builder.go

```diff
@@ -97,8 +97,6 @@ func (builder *StakedAccessNodeBuilder) Initialize() error {
 		return err
 	}
 
-	builder.EnqueueAdminServerInit()
```

Comment:
I see. Is this function still being used? If not, we could just remove

Reply:
The same exact function is being called in scaffold.go.

The problem was that if we call it again in the access node builder, it will be called twice.